### ISSUE TITLE
upgrade cg-busy 4.1.4 to npm

### ISSUE THREAD

**Issue by ghernandez81** [cgross/angular-busy#62]

**Comment by ghernandez81** [comment]
currently in npm it's 4.1.0

**Comment by gabehayes** [comment]
Also, there is a tag missing for `v4.1.4`, so you can't event set a branch/tag target in the interim.

**Comment by cgross** [comment]
Thats because there is no 4.1.4.  The latest ver is 4.1.3.

**Comment by Valko54** [comment]
Please, can you upgrade angular-busy in NPM ? (npm install angular-busy@4.1.3 doesn't exist)

**Comment by axelson** [comment]
+1

**Comment by benoror** [comment]
Hi, I would also benefit from having it updated, let me give you my reason: 

**Support for Webpack + CommonJS Package Manager**

# Motivation

I'm in the process of refactoring an app to support modern JS tooling like NPM, Webpack & ES6.

# Code Change

A small change is needed to support CommonJS:

```
/* commonjs package manager support (eg componentjs) */
if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
  module.exports = 'angular-md5';
}
```
* Source: [ui-router release](https://github.com/angular-ui/ui-router/blob/master/release/angular-ui-router.js#L9)
* Or: [UMD (Universal Module Definition)](https://github.com/umdjs/umd)

# Working Examples

I have done other PRs for some of my dependencies:

* https://github.com/wix/angular-tree-control/pull/152
* https://github.com/gdi2290/angular-md5/issues/8
  * https://github.com/gdi2290/angular-md5/pull/9
* https://github.com/muratcorlu/angular-urlify/pull/1
* https://github.com/tarkanlar/Angular-Urlify/pull/1

# Sources

* [Using Angular 1.x With ES6 and Webpack by Jesus Rodriguez](http://angular-tips.com/blog/2015/06/using-angular-1-dot-x-with-es6-and-webpack/)
* [Webpack & Angular by Shawn McKay ](http://www.shmck.com/webpack-angular-part-3/)
* [Angular and Webpack for Modular Applications by @kentcdodds](https://egghead.io/series/angular-and-webpack-for-modular-applications)
* [NPM for Everything](http://beletsky.net/2015/04/npm-for-everything.html)

**Comment by benoror** [comment]
BTW @gabehayes you can use latest master commit with the following format in **package.json**:

```
"angular-busy": "git+https://github.com/cgross/angular-busy.git"
```

**Comment by myflowpl** [comment]
+1
but i think there should be ```module.exports = 'cgBusy'``` instead of  ```module.exports = 'angular-md5'``` 
then it's compatibile with CommonJS and Angular style (and Webpack)

**Comment by benoror** [comment]
You're right, fixed! Any chance we could se an npm release soon @cgross ?

**Comment by bikegriffith** [comment]
Would love to see this so I don't have to fork and maintain to use in my webpack-driven project
